spec/twitter-ads/creative/promoted_tweet_spec.rb

```diff
@@ -41,6 +41,24 @@
       expect { subject.save }.to raise_error(TwitterAds::ClientError)
     end
 
+    it 'converts params[:tweet_id] to params[:tweet_ids]' do
```

Comment:
I believe this converts `params[:tweet_ids]` to `params[:tweet_id]`

Reply:
Oh really?  
It seems that `TwitterAds::Creative::PromotedTweet#save` set `params[:tweet_ids]` from `params[:tweet_id]` .
(I am following this source code comment.)
https://github.com/twitterdev/twitter-ruby-ads-sdk/blob/master/lib/twitter-ads/creative/promoted_tweet.rb#L49

I have rewritten this description more clearly.